.changeset/gentle-kings-greet.md

```diff
@@ -0,0 +1,11 @@
+---
+"@rocket.chat/meteor": major
+---
+
+Adds a new set of permissions to provide a more granular control for the creation and deletion of rooms within teams
+ - `create-team-channel`: controls the creations of public rooms within teams, it is checked within the team's main room scope and overrides the global `create-c` permission check;
+ - `create-team-group`: controls the creations of private rooms within teams, it is checked within the team's main room scope and overrides the global `create-c` permission check;
```

Comment:
```suggestion
 - `create-team-group`: controls the creations of private rooms within teams, it is checked within the team's main room scope and overrides the global `create-p` permission check;
```

Reply:
I've just improved the changeset overall a bit. Hopefully it's easier to understand the changes now